.deployment/barista/barista-internal.Dockerfile

```diff
@@ -0,0 +1,12 @@
+
+FROM nginx:alpine
+
+COPY .deployment/barista/nginx/nginx.conf /etc/nginx/nginx.conf
+COPY .deployment/barista/nginx/sites-available /etc/nginx/sites-available
+
+RUN ln -s /etc/nginx/sites-available /etc/nginx/sites-enabled
```

Comment:
I have no idea about nginx - but is this supposed to be different - available - enabled?

Reply:
jep in Nginx configurations you store your config for the available sites and then link the one that are currently enabled.